muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/base/material-loader/fields/connect-field.tsx

```diff
@@ -241,10 +241,10 @@ export default class ConnectField extends React.Component<ConnectFieldProps, Con
         }
         return f;
       })
-    }, this.triggerChange)
+    }, triggerChange ? this.triggerChange : null)
   }
   //swapping two counterparts, same as fields
-  swapCounterpart(fielda: FieldType, fieldb: FieldType){
+  swapCounterpart(triggerChange: boolean, fielda: FieldType, fieldb: FieldType){
```

Comment:
Shorter way to express this is could be:
```
triggerChange && this.triggerChange
```

Reply:
I don't understand, it's a parameter; why would it be part of "this"?